cut_a_cake.py

```diff
@@ -0,0 +1,16 @@
+def get_recipe_price(prices, optionals=[], **ingredients):
+    cost = 0
+    for i in ingredients:
```

Comment:
Try using a different name instead of "i" that will be more intuitive.
Same for "cost".